third_party/nvfuser/python/__init__.py

```diff
@@ -12,14 +12,26 @@ def __exit__(self, type, value, traceback):
     def definition(self):
         raise NotImplementedError("definition() should be implemented by child class!")
 
-    def execute(self, inputs):
-        # if definition is not previously defined by a context manager try a child class
+    def schedule(self):
+        raise NotImplementedError("schedule() should be implemented by child class!")
+
+    def execute(self, inputs, override_user_schedule=False):
```

Comment:
Is `override_user_schedule` used here because we can't schedule an `executable` kernel to run with user schedule at the moment?
Nitpicking on debugging API, should we make it a kwarg? I feel that's a little bit more future-proof?

Reply:
I think you are right.  Moving to `**kwargs` makes sense!

Reply:
The `override_user_schedule` kwarg is added to allow the user to execute the same definition with the auto-generated schedule as a comparison point, at least, for benchmarking.  Otherwise, the user defined scheduled will always be executed with a given schedule.  It seemed convenient while debugging.

Reply:
`execute()` now uses kwargs.